Libraries/Components/TimePickerAndroid/NativeTimePickerAndroid.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import Platform from '../../Utilities/Platform';
+
+import type {
+  TimePickerOptions,
+  TimePickerResult,
+} from './TimePickerAndroidTypes';
```

Comment:
Unfortunately, all types used by the spec need to be declared in this file. Perhaps we should relocate these types to this file and have files that depend on TimePickerAndroidTypes instead depend on NativeTimePickerAndroid?

Reply:
Sounds good, I did that in https://github.com/facebook/react-native/pull/24897/commits/03b99364f6f752c121fd502c116caee787cb7d4a